src/hypertrace/agent/instrumentation/aws_lambda/__init__.py

```diff
@@ -79,16 +81,23 @@ def _instrumented_lambda_handler_call( # pylint:disable=R0914
                         http_context = lambda_request_context.get('http')
                         span.set_attribute(SpanAttributes.HTTP_METHOD, http_context.get('method', None))
                         span.set_attribute(SpanAttributes.HTTP_SCHEME, http_context.get('protocol', None))
-                        span.set_attribute(SpanAttributes.HTTP_URL, http_context.get('path', None))
-                        span.set_attribute(SpanAttributes.HTTP_TARGET, http_context.get('path', None))
-                        span.set_attribute(SpanAttributes.HTTP_HOST, headers.get('host', None))
-                    if lambda_request_context.get('path', None):
+                        host = headers.get('host', None)
+                        path = http_context.get('path', '')
+                        if host:
```

Comment:
We don't need to set URL as long as we set scheme/host/target attributes per the otel semantic conventions:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-server-definitions

Relevant section:
```
Additional attribute requirements: At least one of the following sets of attributes is required:

http.scheme, http.host, http.target
http.scheme, http.server_name, [net.host.port](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md), http.target
http.scheme, [net.host.name](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md), [net.host.port](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md), http.target
http.url
```

Reply:
Removed http.url - host, target & scheme are correctly being set.